pyevmasm/evmasm.py

```diff
@@ -122,8 +125,283 @@ def __setitem__(self, key, value):
+        0x00: ('STOP', 0, 0, 0, 0, 'Halts execution.'),
+        0x01: ('ADD', 0, 2, 1, 3, 'Addition operation.'),
+        0x02: ('MUL', 0, 2, 1, 5, 'Multiplication operation.'),
+        0x03: ('SUB', 0, 2, 1, 3, 'Subtraction operation.'),
+        0x04: ('DIV', 0, 2, 1, 5, 'Integer division operation.'),
+        0x05: ('SDIV', 0, 2, 1, 5, 'Signed integer division operation (truncated).'),
+        0x06: ('MOD', 0, 2, 1, 5, 'Modulo remainder operation.'),
+        0x07: ('SMOD', 0, 2, 1, 5, 'Signed modulo remainder operation.'),
+        0x08: ('ADDMOD', 0, 3, 1, 8, 'Modulo addition operation.'),
+        0x09: ('MULMOD', 0, 3, 1, 8, 'Modulo multiplication operation.'),
+        0x0a: ('EXP', 0, 2, 1, 10, 'Exponential operation.'),
+        0x0b: ('SIGNEXTEND', 0, 2, 1, 5, "Extend length of two's complement signed integer."),
+        0x10: ('LT', 0, 2, 1, 3, 'Less-than comparision.'),
+        0x11: ('GT', 0, 2, 1, 3, 'Greater-than comparision.'),
+        0x12: ('SLT', 0, 2, 1, 3, 'Signed less-than comparision.'),
+        0x13: ('SGT', 0, 2, 1, 3, 'Signed greater-than comparision.'),
+        0x14: ('EQ', 0, 2, 1, 3, 'Equality comparision.'),
+        0x15: ('ISZERO', 0, 1, 1, 3, 'Simple not operator.'),
+        0x16: ('AND', 0, 2, 1, 3, 'Bitwise AND operation.'),
+        0x17: ('OR', 0, 2, 1, 3, 'Bitwise OR operation.'),
+        0x18: ('XOR', 0, 2, 1, 3, 'Bitwise XOR operation.'),
+        0x19: ('NOT', 0, 1, 1, 3, 'Bitwise NOT operation.'),
+        0x1a: ('BYTE', 0, 2, 1, 3, 'Retrieve single byte from word.'),
+        0x20: ('SHA3', 0, 2, 1, 30, 'Compute Keccak-256 hash.'),
+        0x30: ('ADDRESS', 0, 0, 1, 2, 'Get address of currently executing account     .'),
+        0x31: ('BALANCE', 0, 1, 1, 20, 'Get balance of the given account.'),
+        0x32: ('ORIGIN', 0, 0, 1, 2, 'Get execution origination address.'),
+        0x33: ('CALLER', 0, 0, 1, 2, 'Get caller address.'),
+        0x34: ('CALLVALUE', 0, 0, 1, 2, 'Get deposited value by the instruction/transaction responsible for this execution.'),
+        0x35: ('CALLDATALOAD', 0, 1, 1, 3, 'Get input data of current environment.'),
+        0x36: ('CALLDATASIZE', 0, 0, 1, 2, 'Get size of input data in current environment.'),
+        0x37: ('CALLDATACOPY', 0, 3, 0, 3, 'Copy input data in current environment to memory.'),
+        0x38: ('CODESIZE', 0, 0, 1, 2, 'Get size of code running in current environment.'),
+        0x39: ('CODECOPY', 0, 3, 0, 3, 'Copy code running in current environment to memory.'),
+        0x3a: ('GASPRICE', 0, 0, 1, 2, 'Get price of gas in current environment.'),
+        0x3b: ('EXTCODESIZE', 0, 1, 1, 20, "Get size of an account's code."),
+        0x3c: ('EXTCODECOPY', 0, 4, 0, 20, "Copy an account's code to memory."),
+        0x3d: ('RETURNDATASIZE', 0, 0, 1, 2, 'Get size of output data from the previous call from the current environment'),
+        0x3e: ('RETURNDATACOPY', 0, 3, 0, 3, 'Copy output data from the previous call to memory'),
```

Comment:
```suggestion
        0x3e: ('RETURNDATACOPY', 0, 3, 0, 3, 'Copy output data from the previous call to memory.'),
```

Reply:
+3 / I am wondering where did it came from...